distributor-node/config/docker/config.docker.yml

```diff
@@ -0,0 +1,25 @@
+id: distributor-node-docker
+endpoints:
+  queryNode: http://graphql-server-mnt:4002/graphql
+  joystreamNodeWs: ws://joystream-node:9944
+directories:
+  assets: /data
+  cacheState: /cache
+log:
+  console:
+    level: info
+limits:
+  storage: 100G
+  maxConcurrentStorageNodeDownloads: 100
+  maxConcurrentOutboundConnections: 300
+  outboundRequestsTimeoutMs: 5000
+  pendingDownloadTimeoutSec: 3600
+  maxCachedItemSize: 1G
+intervals:
+  saveCacheState: 60
+  checkStorageNodeResponseTimes: 60
+  cacheCleanup: 60
+port: 3334
```

Comment:
This is already configured on the `giza_staging` branch, I'd like to avoid changing the config here, because the docker setup on `giza` branch is outdated compared to `giza_staging`, so this would cause unnecessary conflicts.